problems/variables/problem.md

```diff
@@ -11,7 +11,13 @@ The above variable is **declared**, but it isn't defined (it does not yet refere
 Here's an example of defining a variable, making it reference a specific value:
 
 ```js
-const example = 'some string'
+let example = 'some string'
+```
+
+When we declare a variable using `let`, we are able to redefine it. If you want to create a variable that never needs to be changed, **declare** it with the keyword `const`. If you try to redefine a `const` variable, you will see an error.
```

Comment:
I've struggled with how to explain this succinctly. I'm not sure saying "a variable that **never** needs to be changed" is accurate and could also confuse people. example:

```javascript
const array = [];
array.push("hello")
```

is valid. However, going into all the specifics would be a lot. 100% open to an update of some kind. Wondering what others might think?

Reply:
Maybe we could steal this from MDN?

"The value of a constant can't be changed through reassignment (i.e. by using the assignment operator), and it can't be redeclared (i.e. through a variable declaration). However, if a constant is an object or array its properties or items can be updated or removed."

Reply:
I think MDN's explanation is better. I am concerned that it is not for beginners, but I believe it is more important not to mislead.